dashboard/app/models/pd/enrollment.rb

```diff
@@ -222,6 +221,19 @@ def self.get_safe_names
     all.map {|enrollment| [enrollment.full_name, enrollment]}
   end
 
+  # TODO: Delete school column
```

Comment:
We just need to make sure that getting rid of the column waits on making use of the existing school data (which is on my plate soon)